wpilibc/src/main/native/cpp/shuffleboard/Shuffleboard.cpp

```diff
@@ -0,0 +1,31 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "frc/shuffleboard/Shuffleboard.h"
+
+#include <networktables/NetworkTableInstance.h>
+
+#include "frc/shuffleboard/ShuffleboardTab.h"
+
+using namespace frc;
+
+detail::ShuffleboardInstance Shuffleboard::root{
+    nt::NetworkTableInstance::GetDefault()};
```

Comment:
We generally avoid static initialization in the library because it races against user program static initialization.  Should this be a singleton getter with a magic static instead?

Reply:
Yea. I did that earlier when merging ShuffleboardRoot and ShuffleboardInstance, but then I was told ShuffleboardRoot had a purpose so I reverted it and never redid the magic static thing.